src/hotspot/share/runtime/javaCalls.cpp

```diff
@@ -118,7 +118,7 @@ JavaCallWrapper::~JavaCallWrapper() {
   // State has been restored now make the anchor frame visible for the profiler.
   // Do this after the transition because this allows us to put an assert
   // the Java->vm transition which checks to see that stack is not walkable
-  // on sparc/ia64 which will catch violations of the resetting of last_Java_frame
+  // which will catch violations of the resetting of last_Java_frame
```

Comment:
Not clear the comment is still valid given we no longer have sparc or ia64 support.

Reply:
should I remove this whole comment then ?

Reply:
Yes please. I can't see anything related to "not walkable" in the thread state transition code. I checked back to Java 8.

Reply:
done.